src/components/SigningModals/TransactionStepper/Step/Illustration.js

```diff
@@ -0,0 +1,101 @@
+import React, { useCallback } from 'react'
+import PropTypes from 'prop-types'
+import { STEP_ERROR, STEP_SUCCESS, STEP_WORKING } from '../stepper-statuses'
+
+const gradients = {
+  [STEP_WORKING]: [
+    ['#98f9ff', '#00c2ff'],
+    ['#01e8f7', '#00c2ff'],
+  ],
+  [STEP_SUCCESS]: [
+    ['#9fe8d2', '#00c2ff'],
+    ['#73d8ce', '#18b9a0'],
+  ],
+  [STEP_ERROR]: [
+    ['#ffa48e', '#f8baab'],
+    ['#fa8e75', '#ef7757'],
+  ],
+}
+
+function Illustration({ status }) {
+  const [backgroundFrom, backgroundTo] = gradients[status][0]
+  const [foregroundFrom, foregroundTo] = gradients[status][1]
+
+  const gradientId = useCallback(id => `${status.description}-${id}`, [status])
+
+  return (
+    <svg
+      viewBox="0 0 70 70"
+      css={`
+        display: block;
+      `}
+    >
+      <defs>
+        <linearGradient
+          id={gradientId('a')}
+          x1="86.03"
+          y1="-2.54"
+          x2="-184.81"
+          y2="207.33"
+          gradientTransform="matrix(1, 0, 0, -1, 0, 72)"
+          gradientUnits="userSpaceOnUse"
+        >
+          <stop offset="0" stopColor={backgroundFrom} />
+          <stop offset="1" stopColor={backgroundTo} />
+        </linearGradient>
+        <linearGradient
+          id={gradientId('b')}
+          x1="59.34"
+          y1="15.79"
+          x2="38.41"
+          y2="19.85"
+          gradientTransform="matrix(1, 0, 0, -1, 0, 72)"
+          gradientUnits="userSpaceOnUse"
+        >
```

Comment:
I chose to inline this as the code is exactly the same in all cases and providing 3 external files with the same data seemed a waste. This has the benefit of providing flicker free transitions because we're not waiting for an external resource to be inserted.